src/explore-education-statistics-admin/src/pages/methodology/adopt-methodology/AdoptMethodologyPage.tsx

```diff
@@ -0,0 +1,59 @@
+import Page from '@admin/components/Page';
+import PageTitle from '@admin/components/PageTitle';
+import publicationService from '@admin/services/publicationService';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import { dashboardRoute } from '@admin/routes/routes';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import AdoptMethodologyForm from '@admin/pages/methodology/adopt-methodology/components/AdoptMethodologyForm';
+import React from 'react';
+import { RouteComponentProps } from 'react-router';
+
+const AdoptMethodologyPage = ({
+  history,
+  match,
+}: RouteComponentProps<{ publicationId: string }>) => {
+  const { publicationId } = match.params;
+
+  const { value, isLoading } = useAsyncHandledRetry(async () => {
+    const [adoptableMethodologies, publication] = await Promise.all([
+      publicationService.getAdoptableMethodologies(publicationId),
+      publicationService.getPublication(publicationId),
+    ]);
+
+    return {
+      adoptableMethodologies,
+      publication,
+    };
+  }, [publicationId]);
+
+  const { adoptableMethodologies, publication } = value ?? {};
+
+  return (
+    <Page wide breadcrumbs={[{ name: 'Adopt a methodology' }]}>
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <PageTitle title="Adopt a methodology" caption={publication?.title} />
+          <LoadingSpinner loading={isLoading}>
+            {adoptableMethodologies && adoptableMethodologies.length > 0 ? (
```

Comment:
No, the ts linter complains about doing that for length checks. I think when this has come up before that you said it might be different in a later version of ts.

Reply:
Cool, fair enough!